recipes/confu_json/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.5)
```

`cxx_Std_20` was added later.... this should be a hook

```suggestion
cmake_minimum_required(VERSION 3.12)
```